package Login;

message Account {

	optional string user_name		= 1;
	optional string password_hash	= 2;
	
	repeated string character_names	= 3;
	
	//Account specific data goes
}

